articles/install-guide/pyinstall.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Develop with Q# + Python
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.python
+---
+
+# Develop with Q# + Python
```

Comment:
```suggestion
# Develop with Q# & Python
```

Reply:
The '+' and the '&' basically mean the same. However, I like more the '+' because it conveys a "modularity feeling" that the '&' does not. And I think modularity is a good feature of Q#, the same Q# code can be called from C# or from Python. Let me know what you think about this before substituting all the '+' with '&'.

Reply:
I'd like some input from @natke and/or @cjgronlund on how this is typically done in docs. I think for a heading this is a grammar thing and not a how we position the language thing, but it'd be nice to know how other languages/SDKs generally do this?

Reply:
I think either is fine. We might want to look at the search data though. I know some symbols don't play well with search queries.